erpnext/regional/report/hmrc_vat/hmrc_vat.py

```
@@ -0,0 +1,868 @@
+		return vat_accounts
+
+	def validate_vat_accounts(self, vat_accounts):
+		accounts = {}
+		for acc in vat_accounts:
+			acc_type = acc.pop("root_type")
+			accounts.setdefault(acc_type, []).append(acc.copy())
+
+		if (
+			not vat_accounts
+			and not frappe.in_test
+			and not frappe.flags.in_migrate
```

Ruff (0.14.7)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)